nixos/release.nix

```diff
@@ -92,15 +92,33 @@ let
   }).config));
 
 
+  cleanConfig = import ./lib/eval-config.nix {
```

Comment:
Hmm, now I see it does create some evaluation problems for tests. Let me investigate.